src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -39,10 +37,10 @@ public class AddCommand extends Command {
             + PREFIX_WARD + "3 "
             + PREFIX_REMARK + "Prevent bed sores. "
             + PREFIX_TAG + "FallRisk";
-
     public static final String MESSAGE_SUCCESS = "New patient added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "A patient with this IC already exists in the address book";
```

Comment:
I think this message is more specific to let the user know why they can't add the person in.

Reply:
Yep will edit in next PR!